### DISCUSSION TITLE
코드 하이라이트 색상 및 적용 범위를 변경하고 싶습니다.

### DISCUSSION THREAD

**Discussion by dhshin05** [pronist/hELLO#122] [Q&A] [answered]
안녕하세요.
스킨은 감사히 쓰고 있습니다.

다만 제가 포스팅하는 주제가 유니티 관련된 내용이 많은데
기본자료형이나, 함수명, 클래스와 더불어
유니티에서 제공하는 GameObject, Transform과 같은 클래스들도 하이라이팅을 하고 싶습니다.

어디를 수정해야 하는지 알 수 있을까요?
감사합니다.
(이미 있는 문의라면 죄송합니다. 이슈를 쭉 훓어도 못 찾겠어서..)

**Comment by pronist** [comment, marked as answer]
@dhshin05 기본적으로 코드하이라이팅은 [highlight.js](https://highlightjs.org/) 를 씁니다. 클래스의 하이라이트 여부는 테마마다 다를 것입니다. 테마 변경은 스킨옵션에서 제공하고 있습니다.
